src/SDKs/PowerBIDedicated/PowerBIDedicated.Tests/Helpers/PowerBIDedicatedTestUtilities.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure;
+using Microsoft.Azure.Management.PowerBIDedicated;
+using Microsoft.Azure.Management.PowerBIDedicated.Models;
+ using Microsoft.Azure.Test.HttpRecorder;
+using Microsoft.Rest;
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Reflection;
+using System.Runtime.Serialization;
+using System.Runtime.Serialization.Json;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+using Microsoft.Azure.Management.ResourceManager;
+using Microsoft.Azure.Management.ResourceManager.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using DedicatedServices.Tests.Helpers;
+using Xunit;
+
+namespace DedicatedServices.Tests.Helpers
+{
+    public static class PowerBIDedicatedTestUtilities
+    {
+        private static HttpClientHandler Handler = null;
+        private static Uri testUri = new Uri("https://api-dogfood.resources.windows-int.net/");
+
+        // These should be filled in only if test tenant is true
+#if DNX451
```

Comment:
@selasagi we do not support DNX target anymore. Please use FullNetFx (which is targeting full desktop version) or everything else just targets .NET standard 1.4